src/Extension/Tasks/ScssLinterTask.yaml

```diff
@@ -0,0 +1,25 @@
+---
+id: 'validation:scss:lint'
```

Comment:
What do you think about rename to `validation:scss:codestyle`?

Reply:
It will be more consistent this way, nice idea